substrate/docs/deprecation-process.md

```diff
@@ -0,0 +1,70 @@
+# Deprecation Process
+
+This deprecation process makes sense while we don’t use [SemVer](https://semver.org/).
+After that, this process will most likely change.
+As deprecation and removal of existing features can happen on any release, we need to be mindful that external builders
+could be impacted by the changes we make.
+The deprecation process tries to mitigate this impact, while still keeping the developer experience, the DevEx, as
+smooth as possible.
+
+To start a deprecation process a new issue with the label `I11-deprecation` needs to be created for correct tracking.
+Then these are the actions to take:
```

Comment:
Can `I11-deprecation` be used on PR?

Reply:
ok, thanks for clarifying. I asked because doc says, that I11 can be used on issue/pr:
`I11-deprecation`: The current issue/pr is, or should be, part of a deprecation process.`

Reply:
Mmm I think that's a documentation error, @the-right-joyce could you please clarify on that one?

Reply:
hi, can't remember why it's on `I*`, will move it to `T*`

Reply:
@juangirini 
updated: T15-deprecation